index.bs

```diff
@@ -65,16 +65,19 @@ beyond those described in the Generic Sensor API [[!GENERIC-SENSOR]].
 Model {#model}
 =====
 
-The orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
+The device orientation sensors's associated {{Sensor}} subclass is the {{DeviceOrientationSensor}} class.
 
-Orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
-of [=low-level=] motion sensors (accelerometer, gyroscope, magnetometer).
+The device orientation sensor is a [=high-level=] sensor which is created through [=sensor-fusion=]
+of the [=low-level=] motion sensors (accelerometer, gyroscope). The type of fusion (complementary
```

Comment:
I guess that the advantage of the "rotation sensor" in comparison with "geomagnetic rotation sensor" is that the gravity vector is more stable. Ie. instead of isolating gravity from say a accelerometer using a low pass filter, you can get a more precise reading using the z component of a complementary filter between accelerometer and gyroscope.

The disadvantage is draining the battery quicker, for a bit of extra precision.